### PR TITLE
Leviathan shield effect scale

### PR DESCRIPTION
**Pull request by HellDragon-HK**

### CODE REVIEW

**Comment by GreatEmerald** [comment]
You should merge #97 with this one, lest they go out of sync.

**Comment by HellDragon-HK** [comment]
How, it says a file of this name already exists use another name

**Comment by GreatEmerald** [comment]
Close the other one, go to your branch (patch-29), edit the other file the same way you did in the other pull request.

**Comment by GreatEmerald** [comment]
Thanks. But just to be sure, are you certain that X should be 0.7 and not 1.0? It doesn't sound right to me.

**Comment by HellDragon-HK** [comment]
Using the screens I got from UT3 of bots using the shields and the old drawscale yes....using the new UT3 drawscale...probably not accurate now but it should be right for the smaller drawscale currently in use

**Comment by GreatEmerald** [comment]
OK, would be nice if someone could confirm (perhaps also update for the UT3 DrawScale values).

**Comment by HellDragon-HK** [comment]
Working on getting UT3 Drawscale updates, all I've got left of Axon is Stealthbender, Hellfire SPMA, and Leviathan to go I believe

**Comment by GreatEmerald** [comment]
Huh, now it's even more skewed compared to the original mesh?..

**Comment by GreatEmerald** [comment]
Also there's an extra `+` in the code...

**Comment by HellDragon-HK** [comment]
What do you mean? In-game it's larger than it was before.

**Comment by HellDragon-HK** [comment]
I see the extra plus and got it I think

btw the Levi shields (specifically Levi) are not actually blocking anything even if I go back to all the original un-edited .uc files.

**Comment by GreatEmerald** [comment]
Since we've determined that the Leviathan shields use a different mesh altogether, it might be better to port that one instead of reusing the Paladin shield mesh. I'm not sure what the difference is, however. Some comparison screenshots of the two could be useful.

**Comment by HellDragon-HK** [comment]
https://github.com/GreatEmerald/UT3Vehicles/pull/74
https://github.com/GreatEmerald/UT3Vehicles/pull/85#issuecomment-354654817

From the screens I already have I'd say the difference is the middle on the Levi shield is open to be able to fire through it unlike the Paladin which rebounds it's shot, don't think there is any other difference

**Comment by GreatEmerald** [comment]
Yea, that's a good point, but also check in the UT3 editor how the two meshes compare (also including size and shape).

Oh yea, and please merge #74 with this pull request, it's literally the same issue.

**Comment by HellDragon-HK** [comment]
Alright then, if I can find them that is, UT3 editor isn't exactly easy to locate things.

**Comment by HellDragon-HK** [comment]
@GreatEmerald Here you go
ss_001
ss_002
ss_003
ss_004

**Comment by GreatEmerald** [comment]
OK, then in the mean while we can use the same mesh for both. I'll add collision notes to the issue.

Then the scale should correspond to the one in UT3: 1.0 here for the Leviathan and 2.2 for the Paladin, on all three axes.

**Comment by HellDragon-HK** [comment]
Where did you find that, using the vehicles drawscale isn't even large enough so how does that work?

**Comment by GreatEmerald** [comment]
I mentioned that here: https://github.com/GreatEmerald/UT3Vehicles/pull/74#issuecomment-354604553

**Comment by HellDragon-HK** [comment]
1.0 looks too small to me but if that's what it's supposed to be....

Still doesn't stop anything however, AVRiLs look like they are going straight through and I was also taking a little health damage while on the laser turret from Raptor plasma

**Comment by GreatEmerald** [comment]
To make sure that the two meshes in UT3 have the same scale, you could also try placing them in a level side by side.

Hm, I'm not seeing where the collision wouldn't match, since these shields extend the Paladin shield. Oh well, we can figure that out later.

**Comment by HellDragon-HK** [comment]
I don't know enough about the editors to do that, UT2004 I might be able to figure it out but UT3 not likely, it's already complaining about 'access denied' with just even looking at things and trying to load a map is a 'you can't edit already cooked files'

**Comment by GreatEmerald** [comment]
I think that warning is about saving, it should still allow you to try things out. Is there a "place static mesh here" if you right-click on a surface in UT3Ed?

**Comment by HellDragon-HK** [comment]
Don't see a 'Place Static Mesh Here' option, there are right click options but nothing direct, I have 0% knowledge of map editing to understand what any of it does

File: Classes/UT3LeviathanShieldEffectBlue.uc [hunk 1]
```diff
@@ -18,7 +18,7 @@ DefaultProperties
         CoordinateSystem=PTCS_Relative
         MaxParticles=1
         UniformSize=false
-        StartSizeRange=(X=(Min=0.6,Max=0.6),Y=(Min=0.75,Max=0.75),Z=(Min=1.0,Max=1.0))
+        StartSizeRange=(X=(Min=0.7,Max=0.7),Y=(Min=1.4,Max=1.4),Z=(Min=1.6,Max=1.6))
```
Comment (GreatEmerald) [inline Classes/UT3LeviathanShieldEffectBlue.uc]: This is out of sync with the rest.

File: Classes/UT3LeviathanShieldEffectRed.uc [hunk 2]
```diff
@@ -18,7 +18,8 @@ DefaultProperties
         CoordinateSystem=PTCS_Relative
         MaxParticles=1
         UniformSize=false
-        StartSizeRange=(X=(Min=0.6,Max=0.6),Y=(Min=0.75,Max=0.75),Z=(Min=1.0,Max=1.0))
+        StartSizeRange=(X=(Min=1.0,Max=1.0),Y=(Min=1.0,Max=1.0),Z=(Min=1.0,Max=1.0))
+        //StartSizeRange=(X=(Min=0.7,Max=0.7),Y=(Min=1.4,Max=1.4),Z=(Min=1.6,Max=1.6))
```
Comment (GreatEmerald) [inline Classes/UT3LeviathanShieldEffectRed.uc]: No need to keep the previous testing values around as comments.

**Comment by GreatEmerald** [comment]
Or is there a binary requirement to begin with? `UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_ShieldActivate01` is now used, but that was always there, right?

**Comment by GreatEmerald** [comment]
Hm, you also mentioned the DrawScale update here, so what's the status of that?

File: Classes/UT3LeviathanShieldEffectBlue.uc (hunk 1 above)
Comment (HellDragon-HK) [reply Classes/UT3LeviathanShieldEffectBlue.uc]: Argh so I did miss one

File: Classes/UT3LeviathanShieldEffectRed.uc (hunk 2 above)
Comment (HellDragon-HK) [reply Classes/UT3LeviathanShieldEffectRed.uc]: I'd like to keep that as to me those values are more correct, the 1.0 just seems small but that's just my opinion and possibly an optical illusion as I will say on the hostile side of the Levi shield it doesn't look as big for some reason as it does when you're driving.....odd I agree

**Comment by HellDragon-HK** [comment]
Yeah the only binary here is the ukx

**Comment by GreatEmerald** [comment]
OK, I'll accept this because it's better than what's in master at the moment. We can tweak it further (and figure out what's going on with the collision) later.

**Pull request merged**